js/react_native/android/src/androidTest/java/ai/onnxruntime/reactnative/OnnxruntimeModuleTest.java

```diff
@@ -125,4 +126,116 @@ public void onnxruntime_module() throws Exception {
       mockSession.finishMocking();
     }
   }
+
+  @Test
+  public void throwWrongSizeInput() {
+    MockitoSession mockSession = mockitoSession().mockStatic(Arguments.class).startMocking();
+    try {
+      when(Arguments.createMap()).thenAnswer(i -> new JavaOnlyMap());
+      when(Arguments.createArray()).thenAnswer(i -> new JavaOnlyArray());
+
+      OnnxruntimeModule ortModule = new OnnxruntimeModule(reactContext);
+
+      InputStream modelStream =
```

Comment:
>nputStream modelStream = [](http://example.com/codeflow?start=7&length=24)

I think the stream must be closed in the finally clause or in a try() clause. I think they are AutoClosable

Reply:
Ok. I mirrored the code from the other react tests as I don't have an environment setup to let me run them.